modules/ROOT/pages/salesforce/salesforce-connector-100-studio.adoc

```diff
@@ -0,0 +1,276 @@
+
+=== Add the Connector Using Exchange
+
+. In Studio, create a Mule project.
+. Click the Exchange icon *(X)* in the upper-left of the Studio task bar.
+. In Exchange, click *Login* and supply your Anypoint Platform username and password.
+. In Exchange, search for "Salesforce".
+. Select the connector and click *Add to project*.
+. Follow the prompts to install the connector.
+
+=== Add the Connector in Studio
+
+. In Studio, create a Mule project.
+. In the Mule Palette view, click *(X) Search in Exchange*.
+. In *Add Modules to Project*, type "Salesforce" in the search field.
+. Click this connector's name in *Available modules*.
+. Click *Add*.
+. Click *Finish*.
+
+== Configure Salesforce
+
+. Drag a connector operation to the Studio canvas.
++
+To use the Salesforce connector as a message processor, drag one of these operations to the Source area of a flow:
++
+** *On Deleted Object* - Initiate your app when a Salesforce user deletes an object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
+** *On Modified Object* - Initiate your app when a Salesforce user modifies an object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
+** *On New Object* - Initiate your app when a Salesforce user creates a new object. Possible object types are `Contact`, `Lead`, `Account`, `Contracts`, `Opportunity`, or the name of a custom object.
+** *Replay channel listener* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place in the specified channel trigger the flow.
++
+NOTE: A channel or topic must be published to Salesforce before a subscription to the channel can be created.
++
+** *Replay topic listener* - Subscribe to a topic. This trigger provides topic-related notification settings for new events and events stored by Salesforce within the 24-hour retention period. Events that take place on records in the specified topic trigger the flow.
+** *Subscribe channel listener* - Subscribe to a streaming channel. This trigger provides channel-related notification settings for new events that take place after you subscribe.
+** *Subscribe topic listener* - Subscribe to a topic. This trigger provides topic-related notification settings for new events that take place after you subscribe.
++
+IMPORTANT: For the *On New Object* and *On Modified Object* operations, specify a UTC value in the *Since* field. If the server is in a different time zone, you must specify a value for the *Since* field; if the *Since* field is left empty, the connector takes the local timestamp. As servers are in datacenters located generally elsewhere from one's location, use of the *Since* field is recommended.
```

Comment:
```suggestion
IMPORTANT: For the *On New Object* and *On Modified Object* operations, you must specify a UTC value in the *Since* field if the server is in a different time zone. If the *Since* field is empty, the connector uses the local timestamp. Specifying a value in the *Since* field is recommended, as servers in datacenters are often located elsewhere from one's location.
```